docs/commands/preview-docs.md

```diff
@@ -1,39 +1,98 @@
 # `preview-docs`
 
-Preview the API reference docs on your local machine.
+## Introduction
 
-If you have a license key, you will have a preview of the premium Redocly API reference docs. The [`login`](./login.md) command also generates a preview of the premium Redocly API reference docs.
+With this command, you can preview the API reference docs on your local machine.
 
-Otherwise, you'll get a preview of Redoc community edition.
+If you have a license key, you will get a preview of the premium Redocly API reference docs. If you don't, you will get a preview of Redoc community edition.
 
+:::success Tip
+To preview docs using the premium Redocly API reference docs, you must authenticate to the API registry first via the [`login`](./login.md) command.
+:::
 
-### `preview-docs` usage
+## Usage
 
+```bash
+openapi preview-docs <entrypoint>
+openapi preview-docs <entrypoint> [--config=<path>] [--port=<value>]
+openapi preview-docs <entrypoint> [--force] [--help] [--version]
+openapi preview-docs <entrypoint> --version
+```
+
+## Options
+
+option                    | type      | required?    | default     | description
+--------------------------|:---------:|:------------:|:-----------:|------------
+`entrypoint`              | `string`  | yes          | -           | Path to the API definition filename that you want to generate preview for. (See [the section below](#entrypoints) for more options)
+`--config`                | `string`  | no           | -           | Specify path to the config file
+`--force`, `-f`           | `boolean` | no           | -           | Generate preview output even when errors occur
+`--help`                  | `boolean` | no           | -           | Show help
+`--port`, `-p`            | `number`  | no           | 8080        | Preview port
+`--skip-decorator`        | `array`   | no           | -           | Ignore certain decorators
+`--skip-preprocessor`     | `array`   | no           | -           | Ignore certain preprocessors
+`--use-community-edition` | `boolean` | no           | -           | Force using Redoc Community Edition for docs preview
+`--version`               | `boolean` | no           | -           | Show version number
+
+## Examples
+
+### Entrypoints
+
+The command behaves differently depending on how you pass an entrypoint to it and whether the [configuration file](#custom-configuration-file) exists. There are the following options:
 
-```shell
-Positionals:
-  entrypoint                                                 [string] [required]
+#### Pass entrypoint directly
```

Comment:
issue: This heading should be more specific in reflecting what we're doing with the entrypoint. We're passing the full path, right? I think we should focus on the differences between these 3 methods:

1 - full path
2 - just the filename 
3 - alias from the config file

and make sure the headings reflect those differences.

Reply:
Not pretty sure about that. Let's better discuss it